slack-base/src/main/java/com/hubspot/slack/client/models/blocks/HeaderIF.java

```diff
@@ -0,0 +1,45 @@
+package com.hubspot.slack.client.models.blocks;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.hubspot.slack.client.models.blocks.objects.Text;
+import com.hubspot.slack.client.models.blocks.objects.TextType;
+import org.immutables.value.Value.Immutable;
+import com.hubspot.immutables.style.HubSpotStyle;
+import org.immutables.value.Value.Check;
+import org.immutables.value.Value.Default;
+import javax.annotation.Nullable;
+import org.immutables.value.Value;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface HeaderIF extends Block {
+  String TYPE = "header";
+
+  @Override
+  @Value.Derived
+  default String getType() {
+    return TYPE;
+  }
+
+  @Default
+  @Nullable
+  @Value.Parameter
+  default Text getText() {
+    return null;
```

Comment:
Given that your precondition check indicates the header can't have a null text block, what you want to do here is remove the entire default block, and make this just:

```java
@Value.Parameter
Text getText();
```

Then you can assume text is not null, which simplifies your check block below.